<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutPage</class>
 <widget class="QWidget" name="CheckoutPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkout</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageCartView">
      <layout class="QVBoxLayout" name="cartLayout">
       <item>
        <widget class="QTableWidget" name="cartTable"/>
       </item>
       <item>
        <widget class="QLabel" name="labelBill">
         <property name="text">
          <string>Your bill is Rs. XXXX. Please pay at the counter or deposit to bank account number 123456.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="backButton">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="confirmOrderButton">
           <property name="text">
            <string>Confirm Order</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageBillView">
      <layout class="QVBoxLayout" name="billLayout">
       <item>
        <widget class="QTableWidget" name="billTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="billButtonsLayout">
         <item>
          <widget class="QPushButton" name="billBackButton">
           <property name="text">
            <string>Back to Cart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="confirmPaymentButton">
           <property name="text">
            <string>Confirm Payment</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
